CMakeLists.txt

```diff
@@ -2,11 +2,19 @@ cmake_minimum_required(VERSION 3.10)
 
 project(Tracy LANGUAGES CXX)
 
+option(TRACY_STATIC "Whether to build Tracy as a static library" NOT ${BUILD_SHARED_LIBS})
```

Comment:
I think `NOT` is not available here.

Reply:
Why would you think that?
I tested on my system and before this change, Tracy would build as a shared lib by default, implying `BUILD_SHARED_LIBS=ON`. And after this change, Tracy still built as a shared lib by default. But if the `NOT` was ignored, I would have expected the default to change to a static lib

Reply:
I set BUILD_SHARED_LIBS=ON and got the following error.

```shell-session
CMake Error at 3rd/tracy/CMakeLists.txt:5 (option):
  option called with incorrect number of arguments: TRACY_STATIC Whether to
  build Tracy as a static library NOT ON
```

Reply:
This is a minimal example. The string `NOT` seems to be evaluated as OFF.

```
cmake_minimum_required(VERSION 3.10)
project(example)

# set(BUILD_SHARED_LIBS OFF)
# set(BUILD_SHARED_LIBS ON)
message("BUILD_SHARED_LIBS=" ${BUILD_SHARED_LIBS})
option(TRACY_STATIC "Whether to build Tracy as a static library" NOT ${BUILD_SHARED_LIBS})
message("TRACY_STATIC=" ${TRACY_STATIC})
```

Reply:
Oh interesting. So apparently `BUILD_SHARED_LIBS` is empty by default, which would explain my test results. 
Alright, I'll hand in a fix shortly. Thanks for bringing this to my attention :+1:

Reply:
https://github.com/wolfpld/tracy/pull/460